stdlib/SuiteSparse/src/cholmod.jl

```diff
@@ -796,15 +796,18 @@ get_perm(FC::FactorComponent) = get_perm(Factor(FC))
 #########################
 
 # Conversion/construction
-function Dense{T}(A::StridedVecOrMat) where T<:VTypes
-    d = allocate_dense(size(A, 1), size(A, 2), stride(A, 2), T)
-    s = unsafe_load(pointer(d))
-    for i in eachindex(A)
-        unsafe_store!(s.x, A[i], i)
+const StridedVecOrMatInclAdjAndTrans = Union{StridedVecOrMat, Adjoint{<:Any, <:StridedVecOrMat}, Transpose{<:Any, <:StridedVecOrMat}}
+function Dense{T}(A::StridedVecOrMatInclAdjAndTrans) where T<:VTypes
+    d = allocate_dense(size(A, 1), size(A, 2), size(A, 1), T)
```

Comment:
This is outside of my area of expertise, but I would rather not change the `stride(A, 2)` we had before to `size(A, 1)`. I guess it was written for a reason.

Reply:
Yeah, that was that small difference between the two `Dense{T}` methods, that still existed. I cannot use `stride(A, 2)` here, because `stride` is not defined for `Adjoint` or `Transpose`... So I see 2 possibilities:

1. Revert the last change and use two `Dense{T}` methods again.
2. Keep it as it is.

What should I do?

Reply:
I thought I tested it because I wasn't sure myself, but I must have messed it up. I now see that there is no `stride` function for `Adjoint`s and `Transpose`s. Sorry for that. Then I'd suggest to have two methods again, just to make sure we don't change behavior of existing methods unnecessarily (beyond the `GC.@preserve` stuff).